app/Http/Controllers/DashboardController.php

```diff
@@ -35,16 +35,16 @@ public function index(): Response
      */
     public function loginTutor(Request $request): RedirectResponse
     {
-        // check if the user is a tutor
-        if (Auth::user()->is_tutor) {
-            $neededPassword = config('app.tutor_password');
+        $neededPassword = '';
 
-            // check if user is admin
-            if (Auth::user()->is_admin) {
-                $neededPassword = config('app.admin_password');
-            }
+        // check we need a password
+        if (Auth::user()->hasRole(['admin'])) {
```

Comment:
Should super admin also have a password?

Reply:
nope